src/pages/resultsView/pathwayMapper/ResultsViewPathwayMapper.tsx

```diff
@@ -184,7 +184,9 @@ export default class ResultsViewPathwayMapper extends React.Component<
         // Alteration data of non-query genes are loaded.
         if (this.isNewStoreReady) {
             this.addGenomicData(this.alterationFrequencyData);
-            this.dismissActiveToasts();
+            if (this.message === this.LOADING_MESSAGE) {
+                this.dismissMessage();
+            }
```

Comment:
Instead of calling `dismissMessage`, can we just decide what message to show here when the new store is ready?